examples/boolean.ftd

```diff
@@ -0,0 +1,1803 @@
+-- boolean $is-monday: false
```

Comment:
all global variables ,component  and function are defined in this file, which we are import in index file and used all varaibles and component ,so that our code line in index file will not be long and for show output on-click, boolean , function we use in index file